boards/shields/x_nucleo_bnrg2a1/x_nucleo_bnrg2a1.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
+	cs-gpios = <&arduino_header 1 GPIO_ACTIVE_LOW>;	/* A1 */
+
+	bluenrg-2@0 {
```

Comment:
Please add a node label in the form:
```
	bluenrg_2_x_nucleo_bnrg2a1: bluenrg-2@0
```

Reply:
Sorry I didn't understand it. Why should a node label be added while it's not referred by the driver?

Reply:
Ok, right. This is not strictly required indeed.